### DISCUSSION TITLE
本人于9月10日上午用141抽0氪拿下飞霄2+1，特发此贴

### DISCUSSION THREAD

**Discussion by oyy050808** [share121/inter-knot#4775] [崩坏：星穹铁道]
![IMG_20240911_071252](https://github.com/user-attachments/assets/15326b40-7de4-4e6a-ad4b-8a7f7a0c50ab)
![IMG_20240911_071321](https://github.com/user-attachments/assets/31e76382-b88c-4e9d-882a-d7f2c83c7658)

**Comment by Robin-txdy** [comment]
恭喜楼主
